exercises/protein-translation/protein_translation_test.py

```diff
@@ -35,25 +37,34 @@ def test_identifies_stop_codons(self):
         for codon in ['UAA', 'UAG', 'UGA']:
             self.assertEqual('STOP', of_codon(codon))
```

Comment:
Per [discussion](https://github.com/exercism/problem-specifications/pull/997/commits/4c5b8e8bf2ee851b1a13e771c7e782ba4b972a78#r149712640) in exercism/problem-specifications#997, I don't think we should test for intermediate results. Thus, all tests should test `of_rna()` (`proteins()` in the spec).